.github/workflows/commit.yml

```diff
@@ -264,3 +222,38 @@ jobs:
           COSIGN_PASSWORD: ${{ secrets.COSIGN_PASSWORD }}
           TOOL: ${{ matrix.tool }}
           ENVIRONMENT: ${{ matrix.environment }}
+  finalize-the-release:
```

Comment:
We break this section out from the `distribute` step because of the matrix. This finalization only happens after all images have been built, pushed to docker hub, and the sbom/vuln scans have been uploaded. We simply download that bundle, checkout the code, create the GitHub release and update the Docker Hub README. These are tasks which only should happen once, not once for the matrix used in the `distribute` step.